app/components/assets/delete-asset.tsx

```diff
@@ -27,7 +27,7 @@ export const DeleteAsset = ({
       <Button
         variant="link"
         data-test-id="deleteAssetButton"
-        className="justify-start rounded-sm px-6 py-3 text-sm font-semibold text-gray-700 outline-none data-[disabled]:pointer-events-none data-[disabled]:opacity-50 hover:bg-slate-100 hover:text-gray-700"
+        className="justify-start rounded-sm px-2 py-3 text-sm font-semibold text-gray-700 outline-none data-[disabled]:pointer-events-none data-[disabled]:opacity-50 hover:bg-slate-100 hover:text-gray-700"
```

Comment:
I think those changes broke the desktop view. The spacing is wrong, the icons are cropped out.
![Screenshot 2023-08-21 at 11 09 38](https://github.com/Shelf-nu/shelf.nu/assets/7840007/b0244d03-5404-4a8b-9aba-134459fc68ff)

Please check how its done on the `location` page. There the icons are styled and sized properly.

Reply:
Fixed it.